File: python/stdlib/string/const.py
def init(node, global_state):
    def tick(value):
        return {"result": node["args"]}

    node["tick"] = tick

def spec(node):
    node["name"] = "String Const"
    node["outputs"]["result"] = "String"
    node["args"] = "Hello"
    node["desc"] = "Returns the arg on the output."

File: lua/stdlib/string/const.lua
local myNode = {}

function myNode.init(node)
  node.tick = function(value)
    result = {}
    result.result = node.args.value
    return result
  end
end

function myNode.spec(node)
  node.name = "String Const"
  node.outputs.result = "String"
  node.args.value = "Hello"
  node.desc = "Returns the arg on the output."
end

return myNode
